modules/javafx.controls/src/main/java/javafx/scene/control/MenuItem.java

```diff
@@ -527,6 +527,16 @@ public ObservableMap<Object, Object> getProperties() {
         return properties;
     }
 
+    /**
+     * Tests if MenuItem has properties.
+     * @return true if node has properties.
```

Comment:
```suggestion
     * Tests if the menu item has properties.
     * @return {@code true} if the menu item has properties
```
Or can use `{@code MenuItem}`

Reply:
I like this, thanks.  I won't be fixing the javadoc in other places - it's not that critical.